resource/ebpf_version.h

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: MIT
 
 #define EBPF_VERSION_MAJOR 0
-#define EBPF_VERSION_MINOR 17
+#define EBPF_VERSION_MINOR 18
```

Comment:
Is this PR targeting #3683 for the next release?

Reply:
No. This PR changes the release policy to update the version of the main branch post forking of the release branch.